### ISSUE TITLE
ng2TelOutput is not working on invalid number

### ISSUE THREAD

**Issue by akvaliya** [gauravsoni119/ng2-tel-input#36]
When user types wrong numbers ng2TelOutput event is not calling.

I am unable to show customer that they have entered wrong number.

Can you please provide suggestion?

**Comment by lippomano** [comment]
Yes, and also when a user deletes the existent phone number the event is not emitted, I am unable to delete it in my model data.

**Comment by lefamp** [comment]
I also just noticed this but it was working before

**Comment by javascriptDC** [comment]
Validation is not still working now. Can you please provide any suggestion?
